backend/src/schema/schema.ts

```diff
@@ -29,7 +29,7 @@ const Mutation = `
     input AddServiceArgs {
         serviceName: String!
         username: String!
-        email: String!
+        email: String
```

Comment:
If we change email to be optional in ServiceUser, be sure to change the types everywhere. I'm fine with this if the github api call doesn't always return an email address for a user.

Reply:
If email is set to private, GitHub API will show the email as null. To match that, the email of a service is now defined as `string | null` (not optional as I incorrectly stated earlier). I edited the original message to clarify that. Email for our local user is unchanged.